src/rollup-config.ts

```diff
@@ -130,6 +135,13 @@ function createInputConfig(
           classPrivateProperty: true,
           exportDefaultFrom: true,
         },
+        transform: {
+          react: {
+            runtime: ['react-jsx', 'react-jsx-dev'].includes(tsCompilerOptions?.jsx)
+              ? 'automatic'
+              : 'classic'
+          }
+        },
```

Comment:
https://github.com/SukkaW/rollup-plugin-swc/blob/eb4c8ae97d649b6f8e932824528bef9b53b018ba/src/index.ts#L101-L107

I am wondering if maybe I should include this in `rollup-plugin-swc3`. Any thoughts?

Reply:
I am going to let `rollup-plugin-swc3` respects `tsconfig` as well.

It is recommended to use `tsconfig: false` if the `bunchee` will supplies all the configuration and wants `swc3` to be headless.

Reply:
Got it! thanks!